index.js

```diff
@@ -22,10 +22,12 @@ module.exports = {
     'eol-last':                    ['error'],
     'eqeqeq':                      ['error'],
     'indent':                      ['error', 2, {SwitchCase: 1}],
+    'key-spacing':                 ['error', {beforeColon: false, afterColon: true}],
```

Comment:
though actually you bring up the important point that `{mode: 'strict'}` is default, should be `minimum` if we want to allow that